### PR TITLE
feat: kalix-spring-parent pom

### PR DESCRIPTION
**Pull request by octonato**
References
-

### CODE REVIEW

**Comment by octonato** [comment]
Hmm, let's not merge it yet. Publishing of the new parent pom won't work yet.

File: publishLocally.sh
Comment (octonato) [inline publishLocally.sh:24]: this is almost the same as in publish-maven.sh

The only difference is that one is doing local install and the other is deploy (publishing) the artifacts.

**Review by octonato — COMMENTED** [review]
Scripts are fixed now. After merge, it should correctly publish the new parent pom.

**Review by ennru — APPROVED** [review]
LGTM.

**Pull request merged**
